src/libs/TransactionUtils.js

```diff
@@ -228,6 +237,20 @@ function getCreated(transaction) {
     return '';
 }
 
+function isReceiptBeingScanned(transaction) {
+    return _.contains([CONST.IOU.RECEIPT_STATE.SCANREADY, CONST.IOU.RECEIPT_STATE.SCANNING], transaction.receipt.state);
+}
+
+/**
+ * Check if the transaction has a non-smartscanning receipt and is missing required fields
+ *
+ * @param {Object} transaction
+ * @returns {Boolean}
+ */
+function hasMissingSmartscanFields(transaction) {
+    return hasReceipt(transaction) && !isReceiptBeingScanned(transaction) && !areModifiedFieldsPopulated(transaction);
```

Comment:
`!isReceiptBeingScanned` involves `OPEN` status as well as `SCANFAILED`, `SCANCOMPLETE`, 
Should we show RBR for "OPEN" status?

Reply:
> !isReceiptBeingScanned involves OPEN status as well as SCANFAILED, SCANCOMPLETE

Could you clarify this? isReceiptBeingScanned checks for SCANFAILED and SCANCOMPLETE

https://github.com/Expensify/App/blob/ebcc8d045695871d8189a6335c6cdfeede657deb/src/libs/TransactionUtils.js#L254-L257

Reply:
According to this function, `!isReceiptBeingScanned` includes every status that isn't in `CONST.IOU.RECEIPT_STATE.SCANREADY, CONST.IOU.RECEIPT_STATE.SCANNING]`.

Reply:
So `hasMissingSmartscanFields` can return true if `OPEN` status because `isReceiptBeingScanned` is false

Reply:
I am not saying `isReceiptBeingScanned` is wrong. It's correct function.

Reply:
What I am concerned is `why should we show RBR when "OPEN" status?`

Reply:
Ah, thanks for clarifying! So I think if we have any missing fields (even if the receipt is in the OPEN state), we would want to show the RBR to highlight any missing fields to the user